keps/sig-storage/1698-generic-ephemeral-volumes/README.md

```diff
@@ -564,8 +569,24 @@ Will be added before the transition to beta.
 
 - Kubernetes 1.19: alpha (tentative)
 
+## Drawbacks
+
+Allowing users to create PVCs indirectly through pod creation is a new
+capability which must be considered in a cluster's security policy. As
+explained in [risks and mitigations above](#risks-and-mitigations),
+the feature has to be configurable on a per-user basis.
```

Comment:
PSP is at best a bandaid, so I suspect that quota is going to have fill the gap.  Can quota apply to storage class?

Reply:
All of the usual storage quotas are also applied when provisioning volumes through the generic ephemeral volume API.